version/couchdb_2_1_1.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash -e
-echo "================= Installing Cassandra 2.1.1 ==================="
+echo "================= Installing Couchdb 2.1.1 ==================="
 echo "[bintray--apache-couchdb-rpm]
 name=bintray--apache-couchdb-rpm
 baseurl=http://apache.bintray.com/couchdb-rpm/el7/x86_64/
 gpgcheck=0
 repo_gpgcheck=0
 enabled=1" > /etc/yum.repos.d/bintray-apache-couchdb-rpm.repo
+#couchbd is failing if its on this path so moving it to other location 
```

Comment:
can we add the link to the page thats suggests this alternative

Reply:
ok sure